### PR TITLE
Initial conversion to shopping cart service

### PR DESCRIPTION
**Pull request by legendofa**

### CODE REVIEW

**Comment by legendofa** [comment]
Requested draft access.

**Comment by legendofa** [comment]
Looks like it works as expected :partying_face:

File: .github/workflows/update-infrastructure-docker.yaml
Comment (nk-coding) [inline .github/workflows/update-infrastructure-docker.yaml:1]: This one will not work until we add the submodule in infrastructure-docker
I recommend you make a PR there which just adds the submodule

Comment (legendofa) [reply .github/workflows/update-infrastructure-docker.yaml:1]: Ok, will do that

File: schemas/wishlist.graphql
Comment (nk-coding) [inline schemas/wishlist.graphql:1]: outdated, please remove

**Review by nk-coding — APPROVED** [review]

**Pull request merged**